java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -369,4 +375,39 @@ CompletableFuture<Long> linsert(
      * }</pre>
      */
     CompletableFuture<Long> lpushx(String key, String[] elements);
+
+    /**
+     * Pops one or more elements from the first non-empty list from the list of provided <code>key
+     * </code>
+     *
+     * @see <a href="https://valkey.io/commands/lmpop/">valkey.io</a> for details.
+     * @param keys The list of provided <code>key</code> names.
+     * @param direction The direction based on which elements are popped from - see {@link
+     *     LmPopDirection}.
+     * @param count The count of elements to be popped and returned.
+     * @return A <code>Map</code> of <code>key</code> names arrays of popped elements
+     * @example
+     *     <pre>{@code
+     * Map<String, String[]> result = client.lmpop(new String[] {"testKey2"}, LmPopDirection.LEFT, 1L).get();
+     * result.forEach((key, value) -> System.out.println("Key: " + key + ", Value: " + Arrays.toString(value)));
+     * }</pre>
+     */
+    CompletableFuture<Map<String, String[]>> lmpop(
+            String[] keys, LmPopDirection direction, long count);
+
+    /**
+     * Pops one element from the first non-empty list from the list of provided <code>key</code>
```

Comment:
Please track for missing periods in the sentences
```suggestion
     * Pops one element from the first non-empty list from the list of provided <code>key</code>.
```